File: C++/StoreClerk.cpp
#include <iostream>
#include <math.h>
using namespace std;
int main(){
    int productquantity;
    double productprice;
    cout << "Input product quantity: ";
    cin >> productquantity;
    cout << "Input product price: ";
    cin >> productprice;
    cout << "Total payment is " << 1.1*(productprice*productquantity);
}

File: Java/StoreClerk.java
package Java;
import java.lang.Math;
import java.util.Scanner;
public class StoreClerk {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Input product price: ");
        double productprice = sc.nextDouble();
        System.out.print("Input product quantity: ");
        int productquantity = sc.nextInt();
        if(productquantity < -(Math.pow(10,9)) && productprice > Math.pow(10,9)){System.out.print("Invalid input!");}
        sc.close();
        double payment = 1.1*(productquantity*productprice);
        System.out.print("Total payment is " );
        System.out.format("%.0f", payment);
    }
}